pkg/datapath/cells.go

```diff
@@ -217,8 +217,6 @@ type datapathParams struct {
 
 	BandwidthManager types.BandwidthManager
 
-	ModulesManager *modules.Manager
```

Comment:
Consider squashing this with the previous commits that removed fields from here.

Reply:
done (squashed all commits that are removing unnecessary deps from the `datapathParams`)